libs/handoverapp/src/lib/config/gardenConfig.ts

```diff
@@ -1,35 +1,43 @@
 import { HandoverPackage } from '@cc-components/handovershared';
+import { useGardenDataSource } from '@cc-components/shared/workspace-config';
+import { useHttpClient } from '@equinor/fusion-framework-react-app/http';
+import { FilterState } from '@equinor/workspace-fusion/filter';
 import { GardenConfig } from '@equinor/workspace-fusion/garden';
-import { ExtendedGardenFields, HandoverCustomGroupByKeys } from '../types';
-import { GardenGroupBy, GardenHeader } from '../ui-garden';
-import { GardenItem } from '../ui-garden/Item';
-import { fieldSettings, getHighlightedColumn, getItemWidth } from '../utils-garden';
+import { GardenHeader, GardenItem } from '../ui-garden';
 
-export const gardenConfig: GardenConfig<
-  HandoverPackage,
-  ExtendedGardenFields,
-  HandoverCustomGroupByKeys,
-  Record<'maxVolume', number>
-> = {
-  initialGrouping: {
-    horizontalGroupingAccessor: 'RFCC',
-    verticalGroupingKeys: [],
-  },
-  getDisplayName: (item) => item.commpkgNo,
-  fieldSettings: fieldSettings,
-  customGroupByKeys: {
-    plannedForecast: 'Forecast',
-    weeklyDaily: 'Weekly',
-  },
-  customViews: {
-    customHeaderView: GardenHeader,
-    customGroupByView: GardenGroupBy,
-    customItemView: GardenItem,
-  },
-  visuals: {
-    calculateItemWidth: getItemWidth,
-    highlightHorizontalColumn: getHighlightedColumn,
-    collapseSubGroupsByDefault: true,
-    rowHeight: 30,
-  },
+export const useGardenConfig = (
+  contextId: string
+): GardenConfig<HandoverPackage, FilterState> => {
+  const client = useHttpClient('cc-app');
+
+  const { getBlockAsync, getGardenMeta, getHeader, getSubgroupItems } =
+    useGardenDataSource({
+      getBlockAsync: (requestArgs) =>
+        client.fetch(`/api/contexts/${contextId}/handover/garden`, requestArgs),
+      getGardenMeta: (requestArgs) =>
+        client.fetch(`/api/contexts/${contextId}/handover/garden-meta`, requestArgs),
+      getHeader: (requestArgs) =>
+        client.fetch(`/api/contexts/${contextId}/handover/garden`, requestArgs),
+      getSubgroupItems: (requestArgs) =>
+        client.fetch(`/api/contexts/${contextId}/handover/subgroup-items`, requestArgs),
+    });
+
+  return {
+    getDisplayName: (item) => item.commissioningPackageNo,
+    initialGrouping: {
+      horizontalGroupingAccessor: 'Priority1',

```

Comment:
This was changed from RFCC previously, is this not available anymore?

Reply:
"validGroupingOptions": [
        "Responsible",
        "Area",
        "Phase",
        "System",
        "Priority2",
        "Priority3"
    ]

Reply:
I think we need to do some backend work then and add some more grouping options